tests/core/pyspec/eth2spec/test/phase0/block_processing/test_process_deposit.py

```diff
@@ -113,35 +113,35 @@ def test_invalid_sig_other_version(spec, state):
     state.eth1_data.deposit_root = root
     state.eth1_data.deposit_count = 1
 
-    yield from run_deposit_processing(spec, state, deposit, validator_index, valid=True, effective=False)
+    yield from run_deposit_processing(spec, state, deposit, validator_index, effective=False)
```

Comment:
do we expect this test to fail?

from the test name `test_invalid_...` I would expect this to be a failure case but it looks like this is a success case

let's either change the name or set `valid=False`

... after looking at the test content, it looks like this is an invalid deposit but is valid with respect to the state transition -- so then I think it aligns w/ the scheme to change the name -- what do you think @hwwhww ?

Reply:
nice catch! I think it's better to rename the "invalid sig" to "incorrect sig" to avoid confusion. updated the test case name.